axis_backend/apps/clients/views/industry_viewset.py

```
@@ -0,0 +1,123 @@
+        """Get direct children of an industry."""
+        children = self.service.get_children(pk)
+        serializer = IndustryListSerializer(children, many=True)
+        return Response(serializer.data)
+
+    @extend_schema(
+        summary="Get industry descendants",
+        tags=["Industries"],
+        responses={200: IndustryListSerializer(many=True)}
+    )
+    @action(detail=True, methods=['get'])
+    def descendants(self, request, pk=None):
```

Ruff (0.14.5)

Unused method argument: `request`

(ARG002)